manifests/image-references

```diff
@@ -134,3 +134,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-tools:latest
+  - name: volume-data-source-validator
+    from:
+      kind: DockerImage
+      name: registry.k8s.io/sig-storage/volume-data-source-validator:v1.0.0
```

Comment:
```suggestion
      name: quay.io/openshift/volume-data-source-validator:latest
```

Reply:
e2e-gcp-csi failed because it got an `ErrImagePull` from `quay.io/openshift/volume-data-source-validator:latest`, maybe just because it wasn't in `image-references`?

Reply:
Overlooked this one, it should be `quay.io/openshift/origin-volume-data-source-validator:latest` I think. However I misunderstood the ART onboarding process and it seems like we have to configure the mirroring to Quay ourselves: https://github.com/openshift/release/pull/67613

I'm not sure that PR is correct but I suppose the CI here will keep failing until the image is mirrored and available.